nsis/src/nsis/tests/NsisExecutableTest.java

```diff
@@ -0,0 +1,31 @@
+package nsis.tests;
+
+import org.junit.Test;
+
+import generic.continues.RethrowContinuesFactory;
+import ghidra.app.util.bin.ByteArrayProvider;
+import ghidra.app.util.bin.format.pe.PortableExecutable.SectionLayout;
+import nsis.file.NsisExecutable;
+import static org.junit.Assert.*;
+
+public class NsisExecutableTest {
+
+	private byte[] mockFile = { (byte) 0xef, (byte) 0xef, (byte) 0xef, (byte) 0xbe, (byte) 0xad, (byte) 0xde, 'N', 'u', 'l', 'l', 's', 'o', 'f', 't', 'I', 'n', 's', 't' };
+	
+	@Test
+	public void testHeaderOffset() {
```

Comment:
Let's call this test `testNsisCreation` and test the entire Nsis creation. Right now all you can test for is the header size. That is fine, we can add more to this test as we go along.

We also want to include the compiled test NSIS files we have. Otherwise you are going to have manually copy the file bytes into here. You can add that in a future PR.

Reply:
Name modified. I will add the testing files in the next PR